app/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,48 @@
+<resources>
+  <string name="app_name">Finde SMS Codes</string>
```

Comment:
this is the app name shown on the phone launcher, please match it with the title.txt file in metadata folder (like "Copy SMS Code", if app title is going to stay English). this is to prevent confusion for new users who install the app.

Reply:
sure, done

Reply:
Funny, the app recognised the "txt" from the notification email i got for your comment as an OTP xD